tests/ViewTest.elm

```diff
@@ -0,0 +1,33 @@
+module ViewTest exposing (suite)
+
+import Test exposing (Test, describe, test)
+import Test.Html.Event as Event
+import Test.Html.Query as Query
+import Test.Html.Selector exposing (containing, tag, text)
+import Main exposing (view, Msg(..))
+
+suite : Test
+suite =
+    describe "The app"
+        [ test "Increase" <|
+            \_ ->
+                let query = view 0 |> Query.fromHtml
```

Comment:
Why cases look different? This one is confusing because of indentation. I assume `let ... in` returns value that's piped later into `Event.simulate`. Don't like it, but could get used to.

Reply:
I've just tried something else and forgot to remove it, I suppose. Thanks!